lib/seattleflu/id3c/api/routes.py

```diff
@@ -131,6 +131,19 @@ def get_scan_positives(session):
     return Response((row[0] + '\n' for row in positives), mimetype="application/x-ndjson")
 
 
+@api_v1.route("/shipping/scan-hcov19-positives-prio", methods = ['GET'])
```

Comment:
I wonder if we want to make this 
`@api_v2.route("/shipping/scan-hcov19-positives", methods = ['GET'])`
since its very similar data, with additional data/grouping. Thoughts?

Reply:
Done and done!